mcs/class/System/System.Net/MacProxy.cs

```diff
@@ -45,12 +51,31 @@ internal class CFObject : IDisposable
 
 		[DllImport (SystemLibrary)]
 		public static extern void dlclose (IntPtr handle);
-
+		
```

Comment:
How about moving these helper / utility classes into some file of their own?  It isn't really specific to "MacProxy" in any way.  We may also already have some of these utility functions once we're doing moving AppleTls code.